packages/netdog/netdog.spec

```
@@ -0,0 +1,92 @@
+%global _cross_first_party 1
+%undefine _debugsource_packages
+
+Name: %{_cross_os}netdog
+Version: 0.1.0
+Release: 0%{?dist}
+Summary: Bottlerocket network configuration helper
+License: Apache-2.0 OR MIT
+URL: https://github.com/bottlerocket-os/bottlerocket
+
+Source2: netdog-tmpfiles.conf
+
+Source10: run-netdog.mount
+Source11: write-network-status.service
+Source12: generate-network-config.service
+
+Source20: 00-resolved.conf
+
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{_cross_os}netdog
+
+%description
+%{summary}.
+
+%package -n %{_cross_os}netdog-systemd-networkd
+Summary: Bottlerocket network configuration helper
+Provides: %{_cross_os}netdog = 2:
+Requires: %{_cross_os}systemd-networkd
+Requires: %{_cross_os}systemd-resolved
+Supplements: %{_cross_os}systemd-networkd
+%description -n %{_cross_os}netdog-systemd-networkd
+%{summary}.
+
+%package -n %{_cross_os}netdog-wicked
+Summary: Bottlerocket network configuration helper
+Provides: %{_cross_os}netdog = 1:
+Requires: %{_cross_os}wicked
+Supplements: %{_cross_os}wicked
+%description -n %{_cross_os}netdog-wicked
+%{summary}.
+
+%prep
+%setup -T -c
+%cargo_prep
+
+%build
+mkdir bin
+
+echo "** Build Netdog Binaries"
+%cargo_build --manifest-path %{_builddir}/sources/Cargo.toml \
+    -p netdog \
+    --target-dir=${HOME}/.cache/networkd
+%cargo_build --manifest-path %{_builddir}/sources/Cargo.toml \
+    -p netdog \
+    --features wicked \
+    --target-dir=${HOME}/.cache/wicked
```

Maybe this helps with self-documenting code: 

```suggestion
    --features wicked \
    --default-features=false \
    --target-dir=${HOME}/.cache/wicked
```